types/ember__application/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for @ember/application 3.0
+// Type definitions for non-npm package @ember/application 3.0
```

Comment:
I'm not sure how the `@ember/*` packages should be handled. They're published to NPM (combined) as [ember-source](https://www.npmjs.com/package/ember-source), but consumed by users as individual `@ember/*` modules.

Reply:
1. This is a machine-readable marker first, human-readable second, and it's *technically* true that `@ember/application` isn't on npm, so I think this should be here. Maybe I should allow an alternate wording, however.
2. What do the import statements look like? Are they literally `import { x } from '@ember/application'`?

Reply:
> Are they literally `import { x } from '@ember/application'`
yes